app/src/org/odk/collect/android/widgets/TriggerWidget.java

```diff
@@ -30,52 +16,55 @@
 import org.javarosa.form.api.FormEntryPrompt;
 
 /**
- * Widget that allows user to scan barcodes and add them to the form.
- * 
- * @author Yaw Anokwa (yanokwa@gmail.com)
+ * Implements xform trigger tags, which are used to display messages on button
+ * presses. This implementation is incomplete, acting like an output tag due to
+ * a lack of message tag handling.
+ *
+ * @author wspride
  */
 public class TriggerWidget extends QuestionWidget {
-
     private CheckBox mTriggerButton;
+    /**
+     * Stores the answer value of this question. Trigger elements shouldn't
+     * have values, so this is contrary to the spec.
+     */
     private TextView mStringAnswer;
-    private boolean mInteractive = true; 
-    private static String mOK = "OK";
 
-    private FormEntryPrompt mPrompt;
+    /**
+     * Shows a checkbox when set.
+     */
+    private boolean mInteractive = true;
```

Comment:
yes, people do use it - it  gives you that little check box that some people like but I don't think we fully consider a question type. I think its a weird legacy thing from odk.collect mostly